subsys/bluetooth/audio/unicast_client.c

```diff
@@ -83,7 +83,6 @@ static struct bt_gatt_subscribe_params avail_ctx_subscribe[CONFIG_BT_MAX_CONN];
  * needs to be done in serial to avoid using the same discover parameters
```

Comment:
this comment should be adjusted now that cp_disc[] has been removed

Reply:
The comment still applies to `loc_cc_disc` and `avail_ctx_cc_disc`